benchmarks/configs/__init__.py

```diff
@@ -15,3 +17,5 @@
 CONFIGS.update(PRETRAININGS)
 CONFIGS.update(YCB)
 CONFIGS.update(MONTY_WORLD)
+CONFIGS.update(DMC_EVAL)
```

Comment:
Thanks for sharing! Is there a reason you are adding these to the tbp.monty repository instead of the monty_lab repo? Since these experiments are npot part of our benchmark test suite I would not add them here in the benchmarks folder. Can we put these configs into this folder https://github.com/thousandbrainsproject/monty_lab/tree/main/monty_capabilities_analysis ? Similar to how we have a bunch of configs here: https://github.com/thousandbrainsproject/monty_lab/tree/main/experiments/configs

Reply:
Well... I ran into this problem with loading models before when dataclasses weren't in `tbp.monty`, but I just successfully loaded a 9-LM `model.pt` while on `main` and had no problem. 

So what *I think* happened was this: about a month ago, I was saving checkpoints during pretraining so I could keep track of how many points were added per epoch. This saved `config.pt` files, and loading those was throwing errors because `config.pt` had pickled everything in a monty experiment. Since the custom dataclasses had to be visible for unpickling, loading `config.pt` without those classes in `tbp.monty` threw an error. I didn't keep detailed notes, but I remember there was a class discovery mechanism in `torch` that wasn't seeing classes defined outside `tbp.monty`.

I had written a custom logger for saving checkpoints, so maybe it was an issue on my end. Like maybe I needed to be dict-ifying the experiment config before saving it.

I think the good news is that I was wrong about not being able to load `model.pt` files. On the other hand, I'll be revisiting checkpointing soon for the rapid learning experiments. While I'm at it, I'll try to nail down what the issue was exactly.

Reply:
Okay interesting, thanks for the extra infos! I'll be curious to hear what you find out.

Reply:
Thanks for sharing those details Scott.

Yeah in terms of making complex data-classes serializable, I'm curious what the best options are, e.g. it might just be worth implementing some custom methods since we don't have that many different types of objects. Something we can continue to keep in mind as part of [the RFC in PR 46](https://github.com/thousandbrainsproject/tbp.monty/pull/46)

Reply:
Ah ok interesting! Yeah this description certainly sounds interesting (form the main 'pydantic' page)

"Models share many similarities with Python's [dataclasses](https://docs.python.org/3/library/dataclasses.html#module-dataclasses), but have been designed with some subtle-yet-important differences that streamline certain workflows related to validation, serialization, and JSON schema generation. You can find more discussion of this in the [Dataclasses](https://docs.pydantic.dev/latest/concepts/dataclasses/) section of the docs."